todo_app/app/views/todo/index.html.erb

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
This file should be in a `/views/todos` directory (plural). That could be why it's not rendering.